flink-libraries/flink-table/src/test/scala/org/apache/flink/table/expressions/validation/ScalarFunctionsValidationTest.scala

```diff
@@ -97,6 +97,13 @@ class ScalarFunctionsValidationTest extends ScalarTypesTestBase {
     testSqlApi("TIMESTAMPADD(YEAR, 1.0, timestamp '2016-02-24 12:42:25')", "2016-06-16")
   }
 
+  // Need to be activated after merge with support of extract microsecond from Calcite's sql parser
+  @Ignore
+  @Test(expected = classOf[ValidationException])
+  def testDecadeWithTime(): Unit = {
+    testSqlApi("EXTRACT(MICROSECOND FROM TIME '12:42:25.345')", "25345000")
```

Comment:
Do not understand. Could you please explain why this code should fail with ValidationException?

Reply:
My mistake. Test is deleted.